SourceX/DiabloUI/diabloui.cpp

```diff
@@ -46,6 +46,8 @@ char *UiTextInput;
 int UiTextInputLen;
 bool textInputActive = true;
 
+static bool prevSgbControllerActive = false;

```

Comment:
Naming seems very weird here. Isn't the `sgb` prefix shorthand for `staticGlobalBool`? Those should always appear at the front of the variable name if at all.